<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecognitionConfigDialog</class>
 <widget class="QDialog" name="RecognitionConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ρυθμίσεις Μοντέλου Αναγνώρισης Προσώπων</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/argus/res/app_icons/view_image.png</normaloff>:/argus/res/app_icons/view_image.png</iconset>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Αποθήκευση</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Αρχεία Μοντέλου (SFace)</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>371</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Αρχείο Μοντέλου</string>
     </property>
     <widget class="QPushButton" name="openSfaceFileButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Άνοιγμα...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>32</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/argus/res/app_icons/view_image.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Εδώ εισάγετε το αρχείο του εκπαιδευμένου μοντέλου αναγνώρισης προσώπων.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="sfaceFPLineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Στη συγκεκριμένη σελίδα μπορείτε να εισάγετε τα απαραίτητα αρχεία για το μοντέλο &quot;SFace&quot;.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Κλείσιμο</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
